src/drivers/SpiNorFlash.h

```diff
@@ -13,11 +13,11 @@ namespace Pinetime {
       SpiNorFlash(SpiNorFlash&&) = delete;
       SpiNorFlash& operator=(SpiNorFlash&&) = delete;
 
-      typedef struct __attribute__((packed)) {
+      struct __attribute__((packed)) Identification{
```

Comment:
There should be a space here.
```suggestion
      struct __attribute__((packed)) Identification {
```